src/main/java/organice/logic/commands/EditCommand.java

```diff
@@ -173,6 +243,13 @@ public boolean equals(Object other) {
         private Phone phone;
         private Nric nric;
         private Type type;
+        private Age age;
+        private Priority priority;
+        private BloodType bloodType;
+        private DoctorInCharge doctorInCharge;
+        private OrganExpiryDate organExpiryDate;
+        private Organ organ;
+        private TissueType tissueType;
 
         public EditPersonDescriptor() {}
```

Comment:
What is the point of this method?

Reply:
For testing i think, they use it in `EditPersonDescriptorBuilder` or something like that, so can build an empty person, its from AB3

Reply:
isnt an empty constructor the default no parameter constructor in java?